packages/edit-widgets/src/components/layout/index.js

```diff
@@ -33,16 +34,12 @@ function Layout( { blockEditorSettings } ) {
 					<FocusReturnProvider>
 						<EditorSkeleton
 							header={ <Header /> }
-							sidebar={ <Sidebar /> }
+							sidebar={ ! isMobile && <Sidebar /> }
 							content={
 								<>
 									<Notices />
 									<div
 										className="edit-widgets-layout__content"
-										role="region"
-										aria-label={ __(
-											'Widgets screen content'
-										) }
 										tabIndex="-1"
```

Comment:
Can we remove this?

Reply:
I didn't because there's an onFocus handler. So it must be important for block unselection.